src/bootstrap/native.rs

```diff
@@ -74,6 +74,11 @@ impl Step for Llvm {
         let done_stamp = out_dir.join("llvm-finished-building");
 
         if done_stamp.exists() {
+            if builder.config.llvm_skip_rebuild {
+                builder.info("Skipping LLVM rebuild");
```

Comment:
We should use language similar to `--keep-stage` here.

```suggestion
                builder.info("Warning: Using a potentially old LLVM; this may not behave well.");
```